[feat] 로그인 및 회원가입 기능 구현

portsungbin opened the pull request:
회원은 본인의 정보만 확인가능한 기능
로그아웃기능
null허용한 값들의 값이 없을때 null로 표시
회원가입시 비밀번호를 한 번 더 인증
이메일을제외한 모든 값은 필수 입력

portsungbin committed:
feat:로그인 및 회원가입 기능 구현

회원은 본인의 정보만 확인가능한 기능
로그아웃기능
null허용한 값들의 값이 없을때 null로 표시
회원가입시 비밀번호를 한 번 더 인증
이메일을제외한 모든 값은 필수 입력

portsungbin committed:
feat:로그인 및 회원가입 기능 구현

회원은 본인의 정보만 확인가능한 기능
로그아웃기능
null허용한 값들의 값이 없을때 null로 표시
회원가입시 비밀번호를 한 번 더 인증
이메일을제외한 모든 값은 필수 입력

sminseong commented on src/controller/Controller.java:
```diff
@@ -0,0 +1,236 @@
+    private Scanner scan = new Scanner(System.in);
+
+    // 회원가입: 필수 항목이 빈 값이면 계속 재입력을 요구함.
+    public boolean signUpUser(String id, String username, String userPassword, String phone,
```
컨벤션 규칙 1-9 함수 파라미터 개수 제한 규칙 위반으로
수정해 주시면 감사하겠습니다!

portsungbin replied:
넵

sminseong commented on src/controller/Controller.java:
```diff
@@ -0,0 +1,236 @@
+    }
+
+    // 회원 로그인
+    public UserDto loginUser(String userid, String userPassword) {
```
불필요한 loginUser 호출 
수정부탁드립니당

portsungbin replied:
넵

portsungbin committed:
feat:로그인 및 회원가입 기능 구현

회원은 본인의 정보만 확인가능한 기능
로그아웃기능
null허용한 값들의 값이 없을때 null로 표시
회원가입시 비밀번호를 한 번 더 인증
이메일을제외한 모든 값은 필수 입력
권한별 로그인 나누기

My-Bang reviewed (approved):
확인했습니다
